lib/muchos/config.py

```diff
@@ -0,0 +1,647 @@
+#
```

Comment:
We no longer use this file, it was replaced by the various files under lib/muchos/config/. It should be deleted and not be a part of your PR.

Reply:
Okay, I have removed this file.